### PR TITLE
Prep for release-0.10

### PR DESCRIPTION
**Pull request by PhilipGough**
## Description

_Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue._

As per title

## Type of change

_What type of changes does your code introduce to the kube-prometheus? Put an `x` in the box that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Changelog entry

_Please put a one-line changelog entry below. Later this will be copied to the changelog file._

```release-note

```

### CODE REVIEW

**Comment by paulfantom** [comment]
Seems like we are failing on k8s 1.23 :thinking:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
+## release-.10 / 2021-12-16
```
Comment (paulfantom) [inline CHANGELOG.md:1]:
```suggestion
## release-0.10 / 2021-12-17
```

I doubt we will merge it today :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+* [FEATURE] Support scraping config-reloader sidecar for Prometheus and AlertManager StatefulSets [#1344](https://github.com/prometheus-operator/kube-prometheus/pull/1344)
+* [FEATURE] Expose prometheus alerting configuration in $.values.prometheus configuration [#1476](https://github.com/prometheus-operator/kube-prometheus/pull/1476)
+* [BUGFIX] Remove deprecated policy/v1beta1 Kubernetes API [#1433](https://github.com/prometheus-operator/kube-prometheus/pull/1433)
+* [BUGFIX] Fix prometheus URL in prometheus-adapter [#1463](https://github.com/prometheus-operator/kube-prometheus/pull/1463)
+* [BUGFIX] Always use proper values scope for namespace in addons [#1518](https://github.com/prometheus-operator/kube-prometheus/pull/1518)
+* [BUGFIX] Fix default empty groups for k8s PrometheusRule [#1534](https://github.com/prometheus-operator/kube-prometheus/pull/1534)
+* [BUGFIX] Grafana 8.3.1, 8.2.7, 8.1.8, and 8.0.7 released with high severity security fix [#1541](https://github.com/prometheus-operator/kube-prometheus/pull/1541)
```
Comment (paulfantom) [inline CHANGELOG.md:27]: We either need to mention all component bumps or neither

Comment (PhilipGough) [reply CHANGELOG.md:27]: i vote for none in that case :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,30 @@
+## release-.10 / 2021-12-17
```
Comment (dgrisonnet) [inline CHANGELOG.md:1]:
```suggestion
## release-0.10 / 2021-12-17
```

File: README.md [hunk 4]
```diff
@@ -91,13 +91,14 @@ $ minikube addons disable metrics-server
...
-| [`release-0.8`](https://github.com/prometheus-operator/kube-prometheus/tree/release-0.8) | ✗               | ✗               | ✔               | ✔               | ✗               | ✗               |
-| [`release-0.9`](https://github.com/prometheus-operator/kube-prometheus/tree/release-0.9) | ✗               | ✗               | ✗               | ✔               | ✔               | ✗               |
-| [`main`](https://github.com/prometheus-operator/kube-prometheus/tree/main)               | ✗               | ✗               | ✗               | ✗               | ✔               | ✔               |
+| kube-prometheus stack                                                                      | Kubernetes 1.19 | Kubernetes 1.20 | Kubernetes 1.21 | Kubernetes 1.22 | Kubernetes 1.23 |
+|--------------------------------------------------------------------------------------------|-----------------|-----------------|-----------------|-----------------|-----------------|
+| [`release-0.6`](https://github.com/prometheus-operator/kube-prometheus/tree/release-0.6)   | ✔               | ✗               | ✗               | ✗               | ✗               |
```
Comment (dgrisonnet) [inline README.md:96]: let's remove release-0.6, we should only support 5 branches

**Review by dgrisonnet — COMMENTED** [review]
Could you also pin kubernetes-grafana to the latest commit and etcd mixins to release-3.5?

**Review by dgrisonnet — APPROVED** [review]
LGTM

**Pull request merged**